core/templates/dev/head/app.js

```diff
@@ -838,6 +838,20 @@ oppia.factory('windowDimensionsService', ['$window', function($window) {
   };
 }]);
 
+// Service for enabling a background mask that leaves navigation visible.
+oppia.factory('backgroundMaskService', function() {
```

Comment:
We're moving to [UpperCamelCase for service names](https://docs.google.com/document/d/1ZDmLN66f53WdDPItFChu9Lr37z0dKoqR-ASX8UM5y60/edit), so let's use BackgroundMaskService here.